### PR TITLE
PSQLADM-434 Release notes and release note tasks for ProxySQL 2.4.4-1.2

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/2.4.4-1.2.md
modified:   docs/release-notes.md
modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/2.4.4-1.2.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+You can [download the original ProxySQL from GitHub](https://github.com/sysown/proxysql/releases). The [ProxySQL documentation](https://proxysql.com/documentation/) provides information on installing and running ProxySQL. 
+
+The [ProxySQL Admin (proxysql-admin)](https://docs.percona.com/proxysql/v2-config.html) tool simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1.x.x](https://docs.percona.com/proxysql/proxysql-v1.html) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2.x.x](https://docs.percona.com/proxysql/psql-functions.html) supports Percona XtraDB Cluster without custom scripts.
+
+The [Percona Scheduler Admin (percona-scheduler-admin)](https://docs.percona.com/proxysql/psa-scheduler.html) tool can automatically perform a failover due to node failures, service degradation, or maintenance.
+
+!!! note
```
Comment (patrickbirch) [inline docs/2.4.4-1.2.md:33]: This note is in several places in the docs. 
Let's remove the note heading and add the text to the PSA paragraph.

File: mkdocs-base.yml [hunk 2]
```diff
@@ -89,7 +89,7 @@ plugins:
 #    with-pdf: # https://github.com/orzih/mkdocs-with-pdf
 #          output_path: '_pdf/PerconaProxySQLAdmintool-2.4.4.pdf'
 #          cover_title: 'Percona ProxySQL Admin tool Documentation'
-#          cover_subtitle: 2.4.4 (October 04, 2022)
+#          cover_subtitle: 2.4.4-1.2 (November 08, 2022)
```
Comment (patrickbirch) [inline mkdocs-base.yml:92]: No PDF for ProxySQL

**Review by mohitj1988 — APPROVED** [review]

File: docs/2.4.4-1.2.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/2.4.4-1.2.md]: Updated

File: mkdocs-base.yml (hunk 2 above)
Comment (alina-derkach-oaza) [reply mkdocs-base.yml]: Updated

**Review by patrickbirch — APPROVED** [review]

**Review by georgelorchpercona — APPROVED** [review]

**Pull request merged**